.travis.yml

```diff
@@ -2,14 +2,32 @@ language: rust
 sudo: false
 cache: cargo
 rust:
+- stable
 - nightly
-- 1.13.0
+- 1.15.0
 matrix:
   include:
   - os: osx
-    rust: 1.13.0
+  - os: osx
+    env: TARGET=x86_64-apple-ios RUST_BACKTRACE=1 RUST_TEST_THREADS=1
 env:
 - RUST_BACKTRACE=1 RUST_TEST_THREADS=1
+before_script:
+- if [ ! -z "$TARGET" ]; then
```

Comment:
Yes, it's not a good name. I have changed it to better describe what it represents. Rust_test_ios requires all of these targets as it's a hard coded list.